src/main/java/org/sagebionetworks/bridge/hibernate/HibernateSurveyElement.java

```diff
@@ -0,0 +1,122 @@
+package org.sagebionetworks.bridge.hibernate;
+
+import java.util.List;
+
+import javax.persistence.Column;
+import javax.persistence.Convert;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import org.sagebionetworks.bridge.models.surveys.SurveyElement;
+import org.sagebionetworks.bridge.models.surveys.SurveyRule;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.collect.ImmutableList;
+
+@Entity
+@Table(name = "SurveyElements")
+public class HibernateSurveyElement implements SurveyElement {
```

Comment:
You are using single table polymorphic mapping in hibernate (all subclasses of survey element are stored in the same table), but where do you tell Hibernate how to store this, or are you using defaults? If so, where do you tell Hibernate that the discriminator column's name is "type" and not "DTYPE" (the default)? See https://docs.jboss.org/hibernate/orm/5.4/userguide/html_single/Hibernate_User_Guide.html#entity-inheritance-single-table